graphdatascience/query_runner/gds_arrow_client.py

```diff
@@ -48,7 +49,7 @@ class GdsArrowClient:
     @staticmethod
     def create(
         arrow_info: ArrowInfo,
-        auth: Optional[tuple[str, str]] = None,
+        arrow_authentication: Optional[ArrowAuthentication] = None,
```

Comment:
this is a somewhat breaking change as GdsArrowClient is intended to be used by others as well.

Could we have a union of ArrowAuthentication and tuple -> build typed version inside create if user passed the tuple

Reply:
also could keep the same name